crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -73,6 +86,10 @@ impl<T: Pod> BufferVec<T> {
         }
     }
 
+    /// Schedules writing of memory from the host to [`RenderDevice`](crate::renderer::RenderDevice).
```

Comment:
What does "schedule" mean in this context?

Reply:
The operation is queued on the `RenderQueue` provided. `RenderQueue` docs say: "This queue is used to enqueue tasks for the GPU to execute asynchronously." 

So, I meant "schedule" in the sense that the operation does not happen right away, but rather is queued. 

Would it be better to say "Queues" instead of "schedules"?

Reply:
> What does "schedule" mean in this context?

The operation is queued on the `RenderQueue` provided. `RenderQueue` docs say: "This queue is used to enqueue tasks for the GPU to execute asynchronously." 

So, I meant "schedule" in the sense that the operation does not happen right away, but rather is queued. 

Would it be better to say "Queues" instead of "schedules"?

Reply:
I would say "Queues" here, and link to the RenderQueue docs in the later description.

Readers should have a nice web of knowledge to follow, showing them how all the parts are connected.